### PR TITLE
Add Tizen support on docs

### PR DESCRIPTION
**Pull request by JoonghyunCho**
This PR updates the additional platform support on the CommunityToolkit and the content links to the .NET MAUI supported versions page.
Tizen support PR https://github.com/CommunityToolkit/Maui/pull/692/ is planned to be merged and released likely on 08 November.

### CODE REVIEW

**Comment by JoonghyunCho** [comment]
@myroot Please see if you want to add more about Tizen specific. Thanks :)

File: docs/maui/index.md [hunk 1]
```diff
@@ -26,8 +26,9 @@ The .NET MAUI Community Toolkit supports the platforms officially supported by [
 * macOS 10.15 or higher, using Mac Catalyst.
 * Windows 11 and Windows 10 version 1809 or higher, using [Windows UI Library (WinUI) 3](/windows/apps/winui/winui3/).
 
-> [!NOTE]
-> While there is support for Tizen (provided by Samsung) in .NET MAUI, the .NET MAUI Community Toolkit does not currently support it.
+## Additional platform support
+
+.NET MAUI Community Toolkit also includes Tizen support, which is provided by Samsung.
```
Comment (bijington) [inline docs/maui/index.md:31]: Should we include this in the section above (Supported versions). Of course that leads me onto the next question, do we have minimum version support for platforms through Tizen?

**Review by bijington — APPROVED** [review]
@JoonghyunCho thank you for this! I just had a minor query about whether Tizen should go in with the other platforms. If we want to keep it where it is then we can proceed 👍

File: docs/maui/index.md (hunk 1 above)
Comment (VladislavAntonyuk) [reply docs/maui/index.md:31]: I agree with @bijington. Let's keep all platforms together.
Minimum version is 6.5 according to csproj

Comment (JoonghyunCho) [reply docs/maui/index.md:31]: Yes, I just tried to align with what is written on [MAUI document](https://learn.microsoft.com/en-us/dotnet/maui/supported-platforms#additional-platform-support), but I think there is no reason to separate on here.

Comment (JoonghyunCho) [reply docs/maui/index.md:31]: I will include Tizen in the section above! :) Thanks for your suggestion.

Comment (JoonghyunCho) [reply docs/maui/index.md]: @VladislavAntonyuk That's true! The tricky part is..  Tizen 7.0 public which will be installed on the products has .net6 in it so basically to public developers 7.0 is the official version that supports .net6.

**Review by bijington — APPROVED** [review]
Awesome thanks for this @JoonghyunCho

**Pull request merged**